git-multimail/README

```diff
@@ -388,6 +394,18 @@ multimailhook.replyToRefchange
     - The value "none", in which case the Reply-To: field will be
       omitted.
 
+multimailhook.refFilterInclusionRegex
+multimailhook.refFilterExclusionRegex
+
+    Regular expressions that can be used to limit refs for which email
+    updates will be sent.  It is an error to specify both an inclusion
+    and an exclusion regex.  If an inclusion regex is specified, emails
+    will only be sent for refs which match this regex.  If an exclusion
+    regex is specified, emails will be sent for all refs except those
+    that match this regex (or that match a predefined regex specific
+    to the environment, such as "^refs/notes" for most environments and
+    "^refs/notes|^refs/changes" for the gerrit environment).
```

Comment:
Generally I think that allowing this setting to be multivalued would be cleaner than allowing space-separated values that are pulled apart and glued together within the code:

    [multimailhook]
        refFilterExclusionRegex = ^refs/notes
        refFilterExclusionRegex = ^refs/changes

Either way you would probably want to glue the regexps together internally into a single regexp using something like

    '|'.join(
        '(?:%s)' % (fragment,)
        for fragment in fragments
        )